common/src/main/java/io/pravega/common/io/serialization/VersionedSerializer.java

```diff
@@ -420,7 +420,7 @@ void deserializeContents(InputStream stream, ReaderType target) throws IOExcepti
      * <code>
      * class Segment { ... }
      *
-     * class SegmentSerializer extends VersionedSerializer.Direct<Segment> {
+     * class SegmentSerializer extends VersionedSerializer.Direct{@literal <Segment>} {
```

Comment:
`&lt;` and `&gt;` might be better in this case, as the text `Segment` is part of the signature like the rest of the string.

Reply:
yeah I thought of that initially but felt that using `&lt;` and `&gt;` makes it less readable for the devs. For now, I replaced `@literal` with `@code`. does that make it better?